react/src/pages/ServingListPage.tsx

```diff
@@ -146,6 +147,7 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
 
   const sortedEndpointList = _.sortBy(modelServiceList?.items, 'name');
 
+  console.log(baiClient);
```

Comment:
Please remove any debugging logs prior to the code review.

Reply:
Sorry.. I missed that